.github/workflows/_provenance.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       actions: read # For getting workflow run info to build provenance
       id-token: write # needed for signing the images
     # need to use non hash version because of: https://github.com/slsa-framework/slsa-github-generator/issues/3498
```

Comment:
This is no longer the case?

Reply:
So this is still the case but the actual link should be https://github.com/slsa-framework/slsa-verifier/issues/12 ? right?

Reply:
No idea. It's all old